src/opera/template/capability.py

```diff
@@ -1,5 +1,5 @@
 class Capability:
     def __init__(self, name, properties, attributes):
         self.name = name
-        self.properties = properties,
+        self.properties = properties
```

Comment:
Marker: this change is linked to the fix in my other comment.

Reply:
That's a good find. The `properties` variable within the `Capability` class is a dict. The unnecessary trailing comma behind  puts the dict into a tuple and this brings only trouble.